### PR TITLE
CBO-280: Expenses additional information

### PR DESCRIPTION
**Pull request by cleargif**
#### What
Adding additional info to travel expenses

#### Ticket

CBO-280

#### Why
So users can give contextual information relating to expenses

### CODE REVIEW

**Comment by colinbruce** [comment]
Looks generally good @cleargif, does the same partial get used to display info for the case_worker?

If not we should probably add the case_worker summary view too...

**Comment by cleargif** [comment]
@colinbruce - no the view is not shared.. I have done the updates

**Review by colinbruce — APPROVED** [review]
👍

File: db/schema.rb [hunk 1]
```diff
@@ -254,6 +255,15 @@
     t.index ["external_user_id"], name: "index_documents_on_external_user_id", using: :btree
   end
 
+  create_table "establishments", force: :cascade do |t|
```
Comment (jsugarman) [inline db/schema.rb:258]: @cleargif do you know if this is a leftover from a migration on another branch

**Pull request merged**